src/api/flaskr/common/config.py

```
@@ -1,47 +1,1030 @@
+        except KeyError:
+            # Return None for missing keys instead of raising
+            return None
 
     def __getattr__(self, key: Any) -> Any:
-        if key in os.environ:
-            return os.environ[key]
-        return self.parent.__getattr__(key)
+        """Get configuration attribute using enhanced config first."""
+        try:
+            return self.enhanced.get(key)
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)